### DISCUSSION TITLE
Semantic UI vs Fomantic-UI

### DISCUSSION THREAD

**Discussion by dharmendra17xcdify** [fomantic/Fomantic-UI#1422] [General] [answered]
# Help Wanted
I want to know how easy is to migrate from semantic ui to fomantic ui

**Comment by ko2in** [comment, marked as answer]
You can do backup files of your current project. Then remove the semantic-ui package, and install fomantic-ui as described in [documentation](https://fomantic-ui.com/introduction/getting-started.html).

If you've any customization for your UI theme, copy from your old semantic-ui project to fomantic-ui. FomanticUI uses the same process as SemanticUI, so you can follow the way you do for SemanticUI to customize the theme and configurations.

And then build your files via gulp and then you're good to go. I think there shouldn't be any problem. All file names for CSS and JS are the same as SemanticUI, so I don't think you would need any further modifications to link to those CSS and JS files in your project.

Of course, test everything after migration and make sure that your project is working properly as before.

There would be slight differences of some UI components between the old SemanticUI and FomanticUI. You can compare by looking on FomanticUI [website](https://fomantic-ui.com/) before migration.

**Comment by lubber-de** [comment]
Fomantic is 99% compatible to Semantic. Major difference is in usage of the rating module, but adjusting rating code is a couple of minutes to do. 
In case you only use the default theme and already make use of cdn, then simply replace your CDN links from semantic to fomantic 

For a quick test if your current application works with fomantic, change your existing links to semantic.min.js and semantic.min.css to

https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.4/dist/semantic.min.js
https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.4/dist/semantic.min.css

**Comment by lubber-de** [comment]
Closing, because explanation how to easily switch are given. Feel free to add more questions in case it's still unclear to you
